packages/lab/src/chat/README.MD

```diff
@@ -0,0 +1,10 @@
+# Chat component
+
+#### Was made during task
+
+[[Frontend] Implement Chat container component](https://superdispatch.atlassian.net/browse/MSG-9)
+[[Frontend] Implement Message component](https://superdispatch.atlassian.net/browse/MSG-10)
```

Comment:
@TravnikovDev this repo is public and could be used by anyone. I think it's better to not use Jira links. Maybe we can add a description directly?